.github/workflows/build.yaml

```
@@ -34,6 +34,17 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  cpp-checks:
+    needs: [cpp-build]
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-post-build-checks.yaml@conda-cpp-post-build-checks
+    with:
+      build_type: nightly
```

```suggestion
      build_type: ${{ inputs.build_type || 'branch' }}
```

This `build.yaml` workflow gets triggered upon multiple events:

- pushes to `branch-*` branches (e.g. PR merges)
- nightly builds (called via the top-level workflow [here](https://github.com/rapidsai/workflows/blob/main/.github/workflows/nightly-pipeline.yaml))

Because of this, we can't hardcode `nightly` otherwise the job won't work correctly for PR merges.

To fix this, we can apply this suggestion, which will make the job work correctly for both events.

Alternatively, we can make this job conditional to only run nightly.

However, if this job uses CPU nodes (which it appears to), then it's probably easier to just run this on both events since we're not constrained on CPU machines.

---

That makes sense! Yeah this check can run on CPU only nodes so having it run on both events is fine.